toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -277,6 +277,7 @@ func (r *RpmRepoCloner) Clone(cloneDeps bool, packagesToClone ...*pkgjson.Packag
 			"--destdir",
 			chrootDownloadDir,
 			pkgName,
+			"--rpmverbosity=yes",
```

Comment:
I think this tooling change should be moved into a separate PR; does not appear directly related to SELinux

Reply:
This change came from when Daniel McIlvaney worked on this branch in the fall.  Now that you highlight it, I'm don't think it's needed at all and can probably be dropped.